Readme.md

```diff
@@ -38,45 +38,45 @@ Why focus on soldering when you can play with the settings instead? With this ap
 - [X] Custom Preset buttons to allow quick change of user defined temperatures (PineSAM extra feature not available directly inside Pinecil).
 - [X] Backend runs locally on all major platforms/OS while the User Interface runs on your favorite browser.
 - [X] Temperature automatically adjusts based on °C or °F setting.
-- [X] Hide Minimum Voltage (per cell) when source is not battery.
+- [X] Power Settings: hide minimum voltage (per cell) when power source is not 3S-6S.
 
 ## Platforms
- | System  | Windows | MacOS | Linux | iOS | Android|
+ | System  | Windows | MacOS | :penguin: Linux | iOS | Android|
```

Comment:
a single emoji makes this line look awkward. perhaps add emojis to each of them or remove it?

Reply:
done

Reply:
I thought I could get better icons from font awesome but as I can't then it's removed